letsdrill.rb

```diff
@@ -1,13 +1,25 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+ if integer >= 90
+ 	"A"
+ elsif integer >= 80
+ 	"B"
+ elsif integer >= 70
+ 	"C"
+ elsif integer >= 60
+ 	"D"
+ else
+ 	"F"
+ end
 end
 
 def shortest_string(array)
-
-  #Put your code here!
-
+	shortest = array.first
```

Comment:
This is an okay way to solve the problem. A better method would be use one of Ruby's built-in enumerable functions.